include/response.h

```diff
@@ -1,17 +1,42 @@
 #ifndef RESPONSE_H
 #define RESPONSE_H
 
-#include <vector>
 #include <string>
-#include <boost/asio.hpp>
-#include "header.h"
+#include <utility>
+#include <vector>
+
+// Represents an HTTP response.
+//
+// Usage:
+//   Response r;
+//   r.SetStatus(RESPONSE_200);
+//   r.SetBody(...);
+//   return r.ToString();
+//
+// Constructed by the RequestHandler, after which the server should call ToString
+// to serialize.
+class Response {
+ public:
+  enum ResponseCode {
+    // Define your HTTP response codes here.
+    OK = 200,
+    NOT_FOUND = 404,
+    INTERNAL_SERVER_ERROR = 500
+  };
+  
+  void SetStatus(const ResponseCode response_code);
+  void AddHeader(const std::string& header_name, const std::string& header_value);
+  void SetBody(const std::string& body);
+  
+  std::string ToString();
+
```

Comment:
Not sure if I should remove the newline, since I copy/pasted it from here https://github.com/UCLA-CS130/webserver-api/blob/master/request_handler.h